apps/mull-ui/src/app/pages/create-event/create-event.spec.tsx

```diff
@@ -37,7 +37,7 @@ describe('CreateEvent', () => {
       fireEvent.click(submitButton);
     });
     const error = utils.container.querySelector('span[class="error-message"]');
-    expect(error).toHaveTextContent('Start Time is required');
+    expect(error).toHaveTextContent('Image File is required');
```

Comment:
If you made the change above then, this should also be changed to 'Image is required'